website/blog/2025-03-05-release-1.17.md

```diff
@@ -0,0 +1,438 @@
+image: /img/blog/podman-desktop-release-1.17/banner.png
+---
+
+import ThemedImage from '@theme/ThemedImage';
+
+Podman Desktop 1.17 Release! 🎉
+
+![podman-desktop-hero-1.17](/img/blog/podman-desktop-release-1.17/banner.png)
+
+Podman Desktop 1.17 is now available! [Click here to download it](/downloads)!
+
+This release brings exciting new features and improvements:
+
+- **New running workflow** 🛠️: Start containers from images in just a few steps.
+- **Registry mirror configuration** 🔄: Simplify registry mirroring setup with a dedicated command.
+- **Smoother kind cluster experience** ☁️: Spin up a Kubernetes cluster effortlessly, even without a pre-installed kind binary.
+- **Podman 5.4** 🚀: Upgrade to the latest Podman engine for enhanced performance and features.
+- **Pods, redefined** 📦: Clear separation between Podman pods and Kubernetes pods for better usability.
+- **Kubernetes experimental mode** ⚡: Change the way on how the resources are collected and monitored.
+
+<!--truncate-->
+
+## Release Details 🔍
+
+This release brings exciting enhancements, making container management smoother than a seal gliding through the waves. Let's dive in!
+
+### Starting a container from scratch
+
+Ever wanted to start a container effortlessly? Now, just enter the image (local or remote), select it from a dropdown, and with a single click, you'll get the perfect startup prompt. In no time, your container is swimming happily in its new environment!
+
+<ThemedImage
+alt="Starting a container from scratch"
+sources={{
+    light: require('./img/podman-desktop-release-1.17/start-container-from-image-light.png').default,
+    dark: require('./img/podman-desktop-release-1.17/start-container-from-image-dark.png').default,
+  }}
+/>
+
+### Mirroring of Registries – No More Fishy Workarounds
+
+When creating a Podman machine from Podman Desktop, we now include a playbook that mounts the host’s `registries.conf` into the VM at `/etc/containers/registries.conf.d/`. To handle this configuration, simply click on 'More Actions' in the Podman machine menu, and you're all set. No more struggling like a seal stuck on an ice floe.
+
+<ThemedImage
+alt="Mirroring of Registries"
+sources={{
+    light: require('./img/podman-desktop-release-1.17/setup-mirror-registry-light.png').default,
+    dark: require('./img/podman-desktop-release-1.17/setup-mirror-registry-dark.png').default,
+  }}
+/>
+
+<ThemedImage
+alt="Mirroring of Registries step 2"
+sources={{
+    light: require('./img/podman-desktop-release-1.17/setup-mirror-registry-step2-light.png').default,
+    dark: require('./img/podman-desktop-release-1.17/setup-mirror-registry-step2-dark.png').default,
+  }}
+/>
+
+### Podman 5.4
+
+A new version of Podman is here. Check out the full details: [Podman 5.4.0 Release Notes](https://github.com/containers/podman/releases/tag/v5.4.0).  
+⚠️ breaking change
+As mentioned in the podman 5.4 release notes, support for Mac/Intel is now on a best-effort basis.  
```

Comment:
suggestion: might be nice to use [admonitions](https://docusaurus.io/docs/markdown-features/admonitions)

 ```suggestion
 :::warning[Breaking change]
 
As mentioned in the podman 5.4 release notes, support for Mac/Intel is now on a best-effort basis.  

:::
```

Reply:
ah yes, I always forgot the admonitions !